incubating/helmfile/step.yaml

```diff
@@ -0,0 +1,236 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: dustinvanbuskirk/helmfile
```

Comment:
The name cannot be under `dustinvanbuskirk` namespace in official steps, please change to `helmfile`

Reply:
Completed.